modules/apps/foundation/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/internal/portlet/action/EditUserMVCActionCommand.java

```diff
@@ -210,8 +210,6 @@ protected void doProcessAction(
 			if (cmd.equals(Constants.ADD)) {
 				user = addUser(actionRequest);
 
-				addSuccessMessage();
```

Comment:
Thanks for catching this, @pei-jung. It was a mistake left over from my testing.